config/state_machine/configs/00_moj/10_foi_case_type/30_foi_trigger_workflow.yml

```diff
@@ -76,6 +76,10 @@
                   if: Workflows::Predicates#not_overturned?
                 extend_for_pit:
                 flag_for_clearance:
+                  # This even defined here is purely because of current implementation for 
+                  # request_further_clearance for non-triggered case otherwise, such event
+                  # shouldn't need to be defined for triggered case
+                  transition_to_using: Workflows::Conditionals#next_state_for_non_trigger
```

Comment:
I find this comment a little unclear perhaps it could be rephrased as:
```
# This is defined here purely because of the current implementation of
# request_further_clearance for a non-triggered case, otherwise such an event
# shouldn't need to be defined for a triggered case
```

Does this make sense to you?